filenames.auto.gni

```diff
@@ -140,6 +140,7 @@ auto_filenames = {
     "lib/common/define-properties.ts",
     "lib/common/ipc-messages.ts",
     "lib/common/web-view-methods.ts",
+    "lib/common/webpack-globals-provider.ts",
```

Comment:
What's this about?

Reply:
churn from another PR - shown up in multiple PRs this week.